cmd/kyma/test/run/cmd.go

```diff
@@ -144,24 +153,41 @@ func matchTestDefinitionNames(testNames []string,
 
 func generateTestsResource(testName string, numberOfExecutions,
 	maxRetries, concurrency int64,
-	testDefinitions []oct.TestDefinition) *oct.ClusterTestSuite {
+	opts ...TestSuiteOption) *oct.ClusterTestSuite {
 
 	octTestDefs := test.NewTestSuite(testName)
-	matchNames := []oct.TestDefReference{}
-	for _, td := range testDefinitions {
-		matchNames = append(matchNames, oct.TestDefReference{
-			Name:      td.GetName(),
-			Namespace: td.GetNamespace(),
-		})
-	}
 	octTestDefs.Spec.MaxRetries = maxRetries
 	octTestDefs.Spec.Concurrency = concurrency
 	octTestDefs.Spec.Count = numberOfExecutions
-	octTestDefs.Spec.Selectors.MatchNames = matchNames
+	for _, opt := range opts {
+		opt(octTestDefs)
+	}
 
 	return octTestDefs
 }
 
+type TestSuiteOption func(suite *oct.ClusterTestSuite)
```

Comment:
Why do you want to export this variable may be small T

Reply:
I refactored the code a bit. now, this was moved to the test package and can be used everywhere.